mkdocs.yml

```
@@ -18,6 +18,7 @@ nav:
   - Home: index.md
   - install.md
   - motivation.md
+  - cookbook.md
   - env_var.md
```

after the suggestion above, this would then become:

```suggestion
  - env_var.md
  - Cookbook: cookbook/
```

(note, I moved it below env_var ... because it looks a bit better once it becomes a folder)